packages/workflow-core/src/lib/utils/definition-validator/extensions-validator.ts

```diff
@@ -21,67 +23,75 @@ export const extensionsValidator = (extensions: WorkflowExtensions, states: Stat
+      validateChildPlugin(plugin as unknown as ISerializableChildPluginParams);
     }
   });
-}
+};
 const validateApiPlugin = (
   plugin: ISerializableHttpPluginParams,
-  states: StateMachine<any, any, any>['states']
+  states: StateMachine<any, any, any>['states'],
 ): void => {
-  validateTransformers(plugin.name, plugin.request.transform)
-  plugin.response.transform && validateTransformers(plugin.name, plugin.response.transform)
+  validateTransformers(plugin.name, plugin.request.transform);
+  plugin.response.transform && validateTransformers(plugin.name, plugin.response.transform);
 
-  validatePluginStateAction(plugin.stateNames, states, plugin.successAction, plugin.errorAction)
+  validatePluginStateAction(plugin.stateNames, states, plugin.successAction, plugin.errorAction);
 };
-const validateWebhookPlugin = (
-  plugin: SerializableWebhookPluginParams,
-): void => {
-  validateTransformers(plugin.name, plugin.request.transform)
+const validateWebhookPlugin = (plugin: SerializableWebhookPluginParams): void => {
+  validateTransformers(plugin.name, plugin.request.transform);
 };
 const validateIterativePlugin = (
   plugin: SerializableIterativePluginParams,
-  states: StateMachine<any, any, any>['states']
+  states: StateMachine<any, any, any>['states'],
 ): void => {
-  validateMapping('jmespath', plugin.iterateOn)
-  validatePluginStateAction(plugin.stateNames, states, plugin.successAction, plugin.errorAction)
+  validateMapping('jmespath', plugin.iterateOn);
+  validatePluginStateAction(plugin.stateNames, states, plugin.successAction, plugin.errorAction);
 };
-const validateChildPlugin = (
-  plugin: ISerializableChildPluginParams
-): void => {
+const validateChildPlugin = (plugin: ISerializableChildPluginParams): void => {
   plugin.transformers?.forEach(transform => {
-    validateMapping(transform.transformer as Parameters<typeof validateMapping>[0], transform.mapping as Parameters<typeof validateMapping>[1])
-  })
+    validateMapping(
+      transform.transformer as Parameters<typeof validateMapping>[0],
+      transform.mapping as Parameters<typeof validateMapping>[1],
+    );
+  });
 };
 
-const validateTransformers = (pluginName: string, transformers: SerializableValidatableTransformer['transform']) => {
+const validateTransformers = (
+  pluginName: string,
+  transformers: SerializableValidatableTransformer['transform'],
+) => {
   transformers.forEach(transform => {
     try {
-      if (transform.transformer === 'jmespath') validateMapping('jmespath', transform.mapping as string);
-      if (transform.transformer === 'json-logic') validateMapping('json-logic', transform.mapping as unknown as Record<string, unknown>)
+      if (transform.transformer === 'jmespath')
+        validateMapping('jmespath', transform.mapping as string);
+      if (transform.transformer === 'json-logic')
+        validateMapping('json-logic', transform.mapping as unknown as Record<string, unknown>);
```

Comment:
Add missing curly braces to `if` statements please. Line 82 is a bit hard to read. 🙏🏼